tasks/install-with-package.yml

```diff
@@ -0,0 +1,7 @@
+---
+- name: Install Certbot.
+  package: name=certbot state=present
```

Comment:
@oxyc - I'd do it, yes. We can default any of the latest OSes with a good package setup to use the package install instead of the source install.